src/jupyterlabcontroller/services/lab.py

```diff
@@ -874,7 +878,8 @@ def build_pod_spec(
             restart_policy="OnFailure",
             security_context=V1PodSecurityContext(
                 run_as_non_root=True,
-                fs_group=user.gid,
```

Comment:
`fs_group` has to be set or the user won't have write access to `/tmp`, I think. Although if you've tested and this is not the case, that's good news.

Reply:
```
[athor@athor-nb ~]$ cd /tmp
[athor@athor-nb tmp]$ cat > bar
foo
[athor@athor-nb tmp]$ ls
bar
[athor@athor-nb tmp]$ cat bar
foo
[athor@athor-nb tmp]$ 
```

Reply:
Oh, great. Then yes, by all means let's drop that. I had looked at doing that before because we don't really want `fs_group` mounting semantics, but had convinced myself it broke things when removed.